_data/navbars/security.yml

```diff
@@ -14,6 +14,8 @@ section:
       path: /security/calico-network-policy
     - title: Get started with Calico network policy for OpenStack
       path: /security/network-policy-openstack
+    - title: Calico policy, tutorial
+      path: /security/calico-policy
```

Comment:
Should this be /security/tutorials/calico-policy ?

Reply:
Another merge conflict error but I it is all fixed now! :D